pyproject.toml

```diff
@@ -82,6 +83,7 @@ cx_Oracle = { version = "*", optional = true }
 psycopg2-binary = { version = "*", optional = true }
 pika = { version = "*", optional = true }
 redis = { version = "*", optional = true }
+redpanda = { version = "*", optional = true }
```

Comment:
The `redpanda` lib is not what you think it is. https://pypi.org/project/redpanda/ neither is https://pypi.org/project/red-panda/

You won't need either of these since this is a kafka-compliant container. See my overall review!